src/main/cpp/mechanisms/DragonTale/L4ScoringPositionState.cpp

```diff
@@ -34,32 +33,34 @@ using namespace DragonTaleStates;
 
 /// @class ExampleForwardState
 /// @brief information about the control (open loop, closed loop position, closed loop velocity, etc.) for a mechanism state
-L4ScoringPositionState::L4ScoringPositionState ( std::string stateName,
-        int stateId,
-        DragonTale *mech,
-        RobotIdentifier activeRobotId ) : State ( stateName, stateId ), m_mechanism ( mech ), m_RobotId ( activeRobotId )
+L4ScoringPositionState::L4ScoringPositionState(std::string stateName,
+											   int stateId,
+											   DragonTale *mech,
+											   RobotIdentifier activeRobotId) : State(stateName, stateId), m_mechanism(mech), m_RobotId(activeRobotId)
 {
 }
 
 void L4ScoringPositionState::Init()
 {
-	Logger::GetLogger()->LogData ( LOGGER_LEVEL::PRINT, string ( "ArrivedAt" ), string ( "L4ScoringPositionState" ), string ( "Init" ) );
+	Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("L4ScoringPositionState"), string("Init"));
 
-	if ( m_RobotId == RobotIdentifier::PRACTICE_BOT_9999 )
+	if (m_RobotId == RobotIdentifier::PRACTICE_BOT_9999)
 		InitPRACTICE_BOT9999();
 }
 
 void L4ScoringPositionState::InitPRACTICE_BOT9999()
 {
-	m_mechanism->UpdateTargetCoralPercentOutput ( 0 );
-	m_mechanism->UpdateTargetAlgaePercentOutput ( 0 );
+	m_mechanism->UpdateTargetCoralPercentOutput(m_CoralTarget);
+	m_mechanism->UpdateTargetAlgaePercentOutput(m_AlgaeTarget);
 	m_mechanism->SetElevatorTarget(m_ElevatorLeaderTarget);
-	m_mechanism->SetArmTarget(m_ArmTarget);	
+	m_mechanism->SetArmTarget(m_ArmTarget);
 }
 
 void L4ScoringPositionState::Run()
 {
 	// Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("L4ScoringPositionState"), string("Run"));
+	if (m_mechanism->GetAlgaeSensorState())
```

Comment:
Everywhere where we have this check we also need to add an OR condition to see if we are in manual state. Since we don't know which sensor we lost. This is in Hold, L1-L4, and Net/process, but just do a find all and verify